cli/src/tests/node_test.rs

```diff
@@ -306,6 +306,31 @@ fn test_parent_of_zero_width_node() {
     assert_eq!(parent, script_element);
 }
 
+#[test]
+fn test_next_sibling_of_zero_width_node() {
+    let grammar_json = load_grammar_file(
+        &fixtures_dir()
+            .join("test_grammars")
+            .join("next_sibling_from_zwt")
+            .join("grammar.js"),
+        None,
+    )
+    .unwrap();
+
+    let (parser_name, parser_code) = generate_parser_for_grammar(&grammar_json).unwrap();
+
+    let mut parser = Parser::new();
+    let language = get_test_language(&parser_name, &parser_code, None);
+    parser.set_language(&language).unwrap();
+
+    let tree = parser.parse("abdef", None).unwrap();
+
+    let root_node = tree.root_node();
+    let missing_c = root_node.child(2).unwrap();
```

Comment:
Could you add a pre-condition check that `missing_c` is a missing `c` node?

Reply:
Good point! Does `assert!(missing_c.is_missing());` and `assert_eq!(missing_c.kind(), "c");` cover it?

Reply:
Yeah, that’s perfect